lib/system/excpt.nim

```diff
@@ -267,7 +267,7 @@ when hasSomeStackTrace:
       var
         tempFrames: array[0..127, PFrame] # but better than a threadvar
     const
-      firstCalls = 32
+      firstCalls = 64
```

Comment:
Why?

Reply:
reverted that change and moved it to https://github.com/nim-lang/Nim/pull/14982